package.json

```diff
@@ -105,6 +105,7 @@
     "raw-loader": "0.5.1",
     "react-test-renderer": "15.6.1",
     "sass-lint": "1.12.1",
+    "uglifyjs-webpack-plugin": "^1.1.4",
```

Comment:
Can you give context on this new package? Thx

Reply:
So we have this command called serve that let us serve the package compiled (with auto build and reloading) so that we can use it in a demo implementation (I have a bunch of them). The problem is that this command was broken because the version of this package bundled with webpack is outdated and broken.